src/renderer/src/elements/main-dashboard.ts

```diff
@@ -1157,6 +1216,26 @@ export class MainDashboard extends LitElement {
             </button>
           </sl-tooltip>
         </div>
+        <div class="row center-content" style="margin-bottom: 5px;">
+          <sl-tooltip content="Search" placement="right" hoist>
+            <button
+              class="moss-button"
+              @click=${() => this.openPocket()}
```

Comment:
How about changing this to `togglePocket()`? I'd find it more intuitive if the pocket closes again when clicking the button twice.